### PR TITLE
Fixing Bugs about old content.

### PR DESCRIPTION
**Pull request by Yizhuzhu**

### CODE REVIEW

File: web/yo/app/scripts/controllers/gene.js [hunk 1]
```diff
@@ -3311,13 +3311,23 @@ angular.module('oncokbApp')
         var geneName = data.geneName;
 
         $scope.saveCallback = function(newTreatmentName, oldContent){
```
Comment (zhx828) [inline web/yo/app/scripts/controllers/gene.js]: Can we give a more meaning for name for this function?

File: web/yo/app/scripts/controllers/gene.js [hunk 2]
```diff
@@ -3311,13 +3311,23 @@ angular.module('oncokbApp')
...
-            $scope.treatmentRef.name = newTreatmentName;
             var name_uuid = $scope.treatmentRef.name_uuid;
-            mainUtils.setUUIDInReview(name_uuid);
-            var therapyObject = {
-                'name': newTreatmentName,
-                'status': 'latest'
-            };
+            if(!_.isEmpty($scope.treatmentRef.name)) {
```
Comment (zhx828) [inline web/yo/app/scripts/controllers/gene.js]: Only update the review content when it's brand new treatment?

Comment (Yizhuzhu) [reply web/yo/app/scripts/controllers/gene.js]: Update the review content when there is old treatment name. I move the old treatment name to review content. Then I update treatment name.

File: web/yo/app/scripts/controllers/gene.js [hunk 3]
```diff
@@ -3310,14 +3310,24 @@ angular.module('oncokbApp')
...
-            $scope.treatmentRef.name = newTreatmentName;
+        $scope.saveTherapiesCallback = function(newTreatmentName, oldContent){
             var name_uuid = $scope.treatmentRef.name_uuid;
-            mainUtils.setUUIDInReview(name_uuid);
-            var therapyObject = {
-                'name': newTreatmentName,
-                'status': 'latest'
-            };
+            if(!_.isEmpty($scope.treatmentRef.name)) {
+                if (_.isUndefined($scope.treatmentRef.name_review)) {
```
Comment (victoria34) [inline web/yo/app/scripts/controllers/gene.js]: Hi @Yizhuzhu, I think you can merge these if statements together (line 3316 ~ 3324).
```
if (_.isUndefined($scope.treatmentRef.name_review)) {
       $scope.treatmentRef.name_review = {
                'updatedBy' : $rootScope.me.name,
                'updateTime':  new Date().getTime(),
       };
} else if (_.isUndefined($scope.treatmentRef.name_review.updatedBy) || _.isUndefined($scope.treatmentRef.name_review.updateTime)) {
       $scope.treatmentRef.name_review.updatedBy = $rootScope.me.name;
       $scope.treatmentRef.name_review.updateTime = new Date().getTime();
}

```

**Pull request merged**